apps/web/src/server/views/lpa/review-questionnaire.njk

```diff
@@ -0,0 +1,268 @@
+{% extends "app/layouts/app-wide.layout.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+
+{% from "_includes/macros/make-document-links-for-type.njk" import makeDocumentLinksForType %}
+{% from "_includes/macros/make-appeal-site-html.njk" import makeAppealSiteHtml %}
+
+{%- macro detailsInput(type, idPrefix, labelText, details, checkboxItems) -%}
+	{%- if type === 'textarea' -%}
+		{{ govukCharacterCount({
+			id: idPrefix + '-missing-or-incorrect-reason',
+			name: idPrefix + '-missing-or-incorrect-reason',
+			maxlength: 500,
+			label: { text: labelText },
+			errorMessage: details.error and { text: details.error.msg },
+			spellcheck: true,
+			value: details.value,
+			rows: '3'
+		}) }}
+	{%- elif type === 'checkboxes' -%}
+		{{ govukCheckboxes({
+			idPrefix: idPrefix + '-missing-or-incorrect-reason',
+			name: idPrefix + '-missing-or-incorrect-reason',
```

Comment:
Any reason why we have a `-` in front of the name and id?

Reply:
It's because the isPrefix is concatenated with the "missing or incorrect reason" text, without the `-` you would end up with IDs like `supporting-documentsmissing-or-incorrect-reason`